pkg/source/reconciler/source/resources/receive_adapter_test.go

```diff
@@ -38,6 +39,7 @@ func TestMakeReceiveAdapter(t *testing.T) {
 		Spec: v1beta1.KafkaSourceSpec{
 			Topics:        []string{"topic1,topic2"},
 			ConsumerGroup: "group",
+			InitialOffset: ptr.Int64(-2),
```

Comment:
what's `-2` ?

Reply:
earliest, i will change to use the constants from sarama library